clippy_lints/src/needless_update.rs

```diff
@@ -26,20 +26,9 @@ declare_clippy_lint! {
     "using `Foo { ..base }` when there are no missing fields"
 }
 
-#[derive(Copy, Clone)]
-pub struct Pass;
+declare_lint_pass!(NeedlessUpdate => [NEEDLESS_UPDATE]);
```

Comment:
cc https://github.com/rust-lang/rust-clippy/pull/3968#discussion_r275840720

It seems that it doesn't trigger the `module_name_repetition` lint